apps/AppWithWearable/pubspec.yaml

```diff
@@ -78,7 +80,10 @@ dependencies:
   objectbox: ^4.0.1
   objectbox_flutter_libs: any
   flutter_foreground_task: ^6.5.0
-  deepgram_speech_to_text: ^2.1.1
+  deepgram_speech_to_text:
+    git:
+      url: https://github.com/DamienDeepgram/deepgram_speech_to_text.git
```

Comment:
This is temporary for testing

PR to update the Flutter Deepgram SDK should be merged and this can be reverted
https://github.com/tempo-riz/deepgram_speech_to_text/pull/13

Reply:
Waiting for this PR to go through for merging this

Reply:
@josancamon19 Flutter SDK PR got merged by the way

Reply:
Updated this to the 2.3.1 version of the SDK now that it has been updated